include/Animations.hpp

```diff
@@ -0,0 +1,11 @@
+#ifndef ANIMATIONS_HPP
+# define ANIMATIONS_HPP
+
+namespace Animations
+{
+  static constexpr char IDLE[] = "idle";
+  static constexpr char WALK[] = "Walk";
+  static constexpr char STAND[] = "Stand";
```

Comment:
I think `static` isn't necessary. Yçu are missing `const` however (in C++11 it is implied by `constexpr`, but not in C++14)